Gemfile

```diff
@@ -68,7 +68,7 @@ gem 'acts_as_list', '~> 0.9.17'
 gem 'braintree', '~> 2.93'
 gem 'bugsnag', '~> 6.11'
 gem 'cancancan', '~> 2.3.0'
-gem 'formtastic', '~> 2.3.1'
+gem 'formtastic', '~> 4.0'
```

Comment:
Maybe `4.0.0` just to be on the safe side?...
The next [release](https://github.com/formtastic/formtastic#compatibility) will require Ruby 6.0, and we don't know what version will it be release as.

Reply:
Bundle will not install gems that are incompatible with current ruby version.